commons/src/main/java/org/infinispan/commons/util/concurrent/ConcurrentHashSet.java

```diff
@@ -30,23 +29,23 @@
     protected final ConcurrentMap<E, Object> map;
```

Comment:
To be honest this class should be deprecated as well. There is https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html#newKeySet-- that can be used instead.

Reply:
To clarify I mean the entire `ConcurrentHashSet` class

Reply:
Added as a separate commit & jira for exposure